src/api/user/User.js

```diff
@@ -0,0 +1,47 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2021, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+export default class User {
+    constructor(userId, name) {
+        this.userId = userId;
+        this.name = name;
+
+        this.getUserInfo = this.getUserInfo.bind(this);
+        this.getUserId = this.getUserId.bind(this);
+        this.getName = this.getName.bind(this);
+    }
+
+    getUserInfo() {
```

Comment:
Why does this method exist when we already have `getUserId()` and `getName()`?

Reply:
I think I just added it as a quick way to get all info. I can remove?

Reply:
Yes please. I want to keep the API as minimal as possible initially.

Reply:
Gotcha! Makes much more sense to return the user object, thanks!